ndex2/client.py

```diff
@@ -414,8 +472,46 @@ def update_cx_network(self, cx_stream, network_id):
 
     def get_network_as_cx_stream(self, network_id):
         """
+
         Get the existing network with UUID network_id from the NDEx connection as a CX stream.
 
+	.. code-block:: python
+
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+		import json
+
+		#this method returns the network specified by network_id as a CX byte stream
+		response = my_ndex.get_network_as_cx_stream('INSERT NETWORK UUID HERE')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client